srtp-ekt-diet/draft-ietf-perc-srtp-ekt-diet.xml

```diff
@@ -478,6 +478,9 @@ EKTField = FullEKTField / ShortEKTField / ExtensionEKTField
 
         <t>These functions have the property that D(K, E(K, P)) = ( P
         concatenated with optional padding) for all values of K and P.
+        The code using the decrypted values knows the length of and
+        can remove the padding thus there is no requirement for the
+        decryption operation to remove the padding. 
         Each cipher also has a limit T on the number of times that it
```

Comment:
Tha change looks good, But thinking if we need to include reference to how the code (on the decryption side) knows this length. I see the reason for confusion is, the E(k,p) operation is adding the padding (inside encryption operation) and decryption side knowing the length outside decryption operation.